src/common/mockable.js

```diff
@@ -11,8 +12,9 @@ function writeFormattedFile(file, data) {
 }
 
 const mockable = {
-  createConfigExplorer: createExplorer,
-  clearConfigExplorerCache: clearCache,
+  loadConfig: load,
+  searchConfig: search,
+  clearConfigCache: clearCache,
```

Comment:
Let's name this as `clearExplorerCache` since we consider prettierConfig + EditorConfig as "config" https://github.com/fisker/prettier/blob/e4a74c05f4502dd4ec70495c3130ff08ab088e05/src/index.js#L18